.circleci/config.yml

```diff
@@ -278,6 +278,20 @@ jobs:
                 precache: android
             - install_fastlane:
                 directory: android
+            - run:
+                command: |
+                    sudo apt-get update && sudo apt-get install -y ninja-build
+                name: Install Ninja
+            - run:
+                command: |
+                    wget https://github.com/Kitware/CMake/releases/download/v3.22.1/cmake-3.22.1-Linux-x86_64.tar.gz
+                    tar -xzvf cmake-3.22.1-Linux-x86_64.tar.gz -C $HOME
+                    export PATH=$HOME/ninja:$HOME/cmake-3.22.1-Linux-x86_64/bin:$PATH
+                name: Install CMake
+            - run:
+                command: |
+                    export PATH=$HOME/ninja:$HOME/cmake-3.22.1-Linux-x86_64/bin:$PATH
```

Comment:
I actually thought each "run" step has its own shell. so if you do "export PATH" here it won't be applied in other steps

Reply:
i will test that. i just got that from steffen